lms/static/scripts/frontend_apps/test/index-test.js

```diff
@@ -50,16 +50,20 @@ describe('LMS frontend entry', () => {
     $imports.$restore();
   });
 
-  it('renders root component', () => {
-    init();
+  ['basic-lti-launch', 'email-notifications'].forEach(mode => {
```

Comment:
This test remains unchanged, but I ensured it tests for the two "types" of app modes, those which route is just the mode with `/app/` prefix, and those that have a completely different URL.